manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.Safari
+PackageVersion: 4.42
+PackageLocale: en-US
+Publisher: Apple

```

```suggestion
Publisher: Apple Inc.
```
This must match Control Panel.